ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/bonsai/trielog/NoOpTrieLogManager.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.trie.bonsai.trielog;
+
+import org.hyperledger.besu.datatypes.Hash;
+import org.hyperledger.besu.ethereum.core.BlockHeader;
+import org.hyperledger.besu.ethereum.trie.bonsai.worldview.BonsaiWorldState;
+import org.hyperledger.besu.ethereum.trie.bonsai.worldview.BonsaiWorldStateUpdateAccumulator;
+import org.hyperledger.besu.plugin.services.trielogs.TrieLog;
+
+import java.util.Optional;
+
+public class NoOpTrieLogManager extends TrieLogManager {
+
+  public NoOpTrieLogManager() {
+    super(null, null, 0, null, TrieLogPruner.noOpTrieLogPruner());
+  }
+
+  @SuppressWarnings({"UnsynchronizedOverridesSynchronized", "squid:S3551"})
+  @Override
+  public void saveTrieLog(
```

Comment:
I think this can be synchronized without any problem.  It looks like Simon didn't add synchronized here when he changed the interface.  This is only used in the referencetest worldstate, so I don't think it should cause an issue to make it synchronized